docs/getting-started-guides/mesos.md

```diff
@@ -132,7 +132,7 @@ Update your PATH to more easily run the Kubernetes-Mesos binaries:
 export PATH="$(pwd)/_output/local/go/bin:$PATH"
 ```
 
-Identify your Mesos master: depending on your Mesos installation this is either a `host:port` like `mesos_master:5050` or a ZooKeeper URL like `zk://zookeeper:2181/mesos`.
+Identify your Mesos master: depending on your Mesos installation this is either a `host:port` like `mesos-master:5050` or a ZooKeeper URL like `zk://zookeeper:2181/mesos`.
```

Comment:
`mesos-master` `auth-path` `km-path` `executor-path` `--proxy-path` I verified are defined by mesos with a `-` and the docs are in fact wrong.

In this case mesos-master is purely a user indicator, and in fact they use mesos-master in other places.